etc/avocado/avocado.conf

```diff
@@ -85,3 +85,7 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+[plugins.result]
+# Support for ordering result plugins *only* at this version
```

Comment:
BTW, I forgot to add the `archive.py` plugin to the last commit.  Force pushing that last one.